src/components/pipelines-overview/PipelineRunsNumbersChart.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { useTranslation } from 'react-i18next';
+import { DomainPropType, DomainTuple } from 'victory-core';
+import {
+  Chart,
+  ChartAxis,
+  ChartBar,
+  ChartGroup,
+  ChartThemeColor,
+  ChartVoronoiContainer,
+} from '@patternfly/react-charts';
+import { Card, CardBody, CardTitle } from '@patternfly/react-core';
+import { formatDate, getXaxisValues, parsePrometheusDuration } from './dateTime';
+
+interface PipelinesRunsNumbersChartProps {
+  timespan?: number;
+  domain?: DomainPropType;
+}
+type DomainType = { x?: DomainTuple; y?: DomainTuple };
+
+const PipelinesRunsNumbersChart: React.FC<PipelinesRunsNumbersChartProps> = ({
+  timespan,
+  domain,
+}) => {
+  const { t } = useTranslation('plugin__pipeline-console-plugin');
+  const startTimespan = timespan - parsePrometheusDuration('1d');
+  const endDate = new Date(Date.now()).setHours(0, 0, 0, 0);
+  const startDate = new Date(Date.now() - startTimespan).setHours(0, 0, 0, 0);
+  const { x: domainX, y: domainY } = (domain as DomainType) || {};
+  const domainValue: DomainPropType = {
+    x: domainX || [startDate, endDate],
+    y: domainY || undefined,
+  };
+  const tickValues = getXaxisValues(timespan);
+
+  const chartData = tickValues.map((value) => {
+    return { x: value, y: 100 };
+  });
+
+  if (!domainY) {
+    let minY: number = _.minBy(chartData, 'y')?.y ?? 0;
+    let maxY: number = _.maxBy(chartData, 'y')?.y ?? 0;
+    if (minY === 0 && maxY === 0) {
+      minY = -1;
+      maxY = 1;
+    } else if (minY > 0 && maxY > 0) {
+      minY = 0;
+    } else if (minY < 0 && maxY < 0) {
+      maxY = 0;
+    }
+    domainValue.y = [minY, maxY];
+  }
+
+  const xTickFormat = (d) => formatDate(d);
+  let xAxisStyle: any = {
```

Comment:
can we define type for `xAxisStyle` instead of any?

Reply:
We can't define particular type for this, we can add any style for this. There can be any number of styling we can add for this, so I don't think we can define a particular type for this.

Reply:
okay, then it's fine